src/ui/QuerySummary/QuerySummary.ts

```diff
@@ -189,19 +233,83 @@ export class QuerySummary extends Component {
     }
   }
 
```

Comment:
The new logic is from line 236 to 261.
Line 236 : updateQueryTagsInNoResultsContainer() : makes possible the usage of the queryTag (${query}) in the code editor of the QuerySummary component.
Line 250 : parseQueryTags() : the new logic for parsing the queryTag from a string.